ArduinoSpoof.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#ifndef ARDUINO_BUILD
+
+#include <cstdlib>
+#include <iostream>
+#include <stdint.h>
+
+#define HIGH 1
+#define LOW 0
+#define INPUT 0
+#define OUTPUT 1
+#define INPUT_PULLUP 2
+#define INPUT_PULLDOWN 3
+#define OUTPUT_OPENDRAIN 4
+#define INPUT_DISABLE 5
+
+#define VERBOSE_ARDUINO
+
+#ifdef VERBOSE_ARDUINO
+#define verbose_print(str) std::cout << str << std::endl;
+#else
+#define verbose_print(str)
+#endif // VERBOSE_ARDUINO
+
+void pinMode(uint8_t pin, uint8_t mode) {
+#ifdef VERBOSE_ARDUINO
```

Comment:
If `VERBOSE_ARDUINO` is not set, these functions result in `unused parameter` errors.